### PR TITLE
Rename contract from code to link

### PR DESCRIPTION
**Pull request by cmgrd**
Renamed code to link, to match whats actually being sent in MeteringPointCreated event.

### CODE REVIEW

**Review by Korgath — APPROVED** [review]

File: source/Energinet.DataHub.MeteringPoints.Infrastructure/Integration/Contracts/IntegrationEventContract.proto [hunk 1]
```diff
@@ -104,7 +104,7 @@ message MeteringPointCreated {
 
   string metering_point_id = 1; // Unique identification for metering point
   string gsrn_number = 2; // Business facing metering point identifier
-  string grid_area_code = 3; // Signifies which grid area a metering point belongs to
+  string grid_area_link = 3; // Signifies which grid area a metering point belongs to
```
Comment (BjarkeMeier) [inline source/Energinet.DataHub.MeteringPoints.Infrastructure/Integration/Contracts/IntegrationEventContract.proto:107]: Isn't a grid area link and the id of a grid area link two different things? Could we suffix all these props with "_id" in order to be specific and avoid confusion or mistakes?

File: source/Energinet.DataHub.MeteringPoints.Infrastructure/Integration/IntegrationEvents/CreateMeteringPoint/MeteringPointCreatedEventMessageMapper.cs [hunk 2]
```diff
@@ -31,9 +31,9 @@ protected override IMessage Convert(MeteringPointCreatedEventMessage obj)
                 MeteringPointId = obj.MeteringPointId,
                 GsrnNumber = obj.GsrnNumber,
                 MeteringPointType = obj.GetMeteringPointType(),
-                GridAreaCode = obj.GridAreaId,
-                ToGridAreaCode = obj.ToGrid,
-                FromGridAreaCode = obj.FromGrid,
+                GridAreaLink = obj.GridAreaId,
+                ToGridAreaLink = obj.ToGrid,
+                FromGridAreaLink = obj.FromGrid,
```
Comment (BjarkeMeier) [inline source/Energinet.DataHub.MeteringPoints.Infrastructure/Integration/IntegrationEvents/CreateMeteringPoint/MeteringPointCreatedEventMessageMapper.cs:36]: From this code block, I can infer that `grid area link = grid area id = grid`.
That doesn't look correct to me. But I don't know whether it's code errors or bad inconsistent naming.

Comment (BjarkeMeier) [reply source/Energinet.DataHub.MeteringPoints.Infrastructure/Integration/IntegrationEvents/CreateMeteringPoint/MeteringPointCreatedEventMessageMapper.cs:36]: I'd probably expect to see something similar to

```suggestion
                GridAreaLinkId = obj.GridAreaLinkId,
                ToGridAreaLinkId = obj.ToGridAreaLinkId,
                FromGridAreaLinkId = obj.FromGridAreaLinkId,
```

Comment (cmgrd) [reply source/Energinet.DataHub.MeteringPoints.Infrastructure/Integration/IntegrationEvents/CreateMeteringPoint/MeteringPointCreatedEventMessageMapper.cs:36]: Inconsistent naming.

Comment (cmgrd) [reply source/Energinet.DataHub.MeteringPoints.Infrastructure/Integration/IntegrationEvents/CreateMeteringPoint/MeteringPointCreatedEventMessageMapper.cs]: Fixed

File: source/Energinet.DataHub.MeteringPoints.Infrastructure/Integration/Contracts/IntegrationEventContract.proto (hunk 1 above)
Comment (cmgrd) [reply source/Energinet.DataHub.MeteringPoints.Infrastructure/Integration/Contracts/IntegrationEventContract.proto]: Appended _id

File: source/Energinet.DataHub.MeteringPoints.Infrastructure/Integration/IntegrationEvents/CreateMeteringPoint/MeteringPointCreatedEventMessageMapper.cs [hunk 3]
```diff
@@ -31,9 +31,9 @@ protected override IMessage Convert(MeteringPointCreatedEventMessage obj)
                 MeteringPointId = obj.MeteringPointId,
                 GsrnNumber = obj.GsrnNumber,
                 MeteringPointType = obj.GetMeteringPointType(),
-                GridAreaCode = obj.GridAreaId,
-                ToGridAreaCode = obj.ToGrid,
-                FromGridAreaCode = obj.FromGrid,
+                GridAreaLinkId = obj.GridAreaLinkId,
+                ToGridAreaLinkId = obj.ToGridLinkId,
+                FromGridAreaLinkId = obj.FromGridLinkId,
```
Comment (BjarkeMeier) [inline source/Energinet.DataHub.MeteringPoints.Infrastructure/Integration/IntegrationEvents/CreateMeteringPoint/MeteringPointCreatedEventMessageMapper.cs:36]:
```suggestion
                GridAreaLinkId = obj.GridAreaLinkId,
                ToGridAreaLinkId = obj.ToGridAreaLinkId,
                FromGridAreaLinkId = obj.FromGridAreaLinkId,
```

Comment (cmgrd) [reply source/Energinet.DataHub.MeteringPoints.Infrastructure/Integration/IntegrationEvents/CreateMeteringPoint/MeteringPointCreatedEventMessageMapper.cs]: Fixed

**Review by Korgath — APPROVED** [review]

**Pull request closed without merging**